x/auth/keeper/keeper.go

```diff
@@ -45,9 +45,6 @@ type AccountKeeperI interface {
 	// Fetch the sequence of an account at a specified address.
 	GetSequence(context.Context, sdk.AccAddress) (uint64, error)
 
-	// Fetch the next account number, and increment the internal counter.
-	NextAccountNumber(context.Context) uint64
```

Comment:
instead of removing this can we use is as a wrapper to accounts to avoid the api break? it can be deprecated right away

Reply:
added it back